theme/tutorial.less

```diff
@@ -471,6 +471,50 @@ code.lang-filterblocks {
     margin: 0.5em;
 }
 
+
+/******************************
+    Immersive Reader button
+******************************/
+#mainmenu .immersive-reader-button.ui.item,
+#mainmenu .immersive-reader-button.ui.item:focus,
+.hc #mainmenu .immersive-reader-button.ui.item:focus {
+    background-image: @immersiveReaderLightIcon;
+    width: 1.8rem;
+    height: 1.8rem;
+    background-repeat: no-repeat;
+    background-size: contain;
+    margin-top: .4rem;
+}
+
+.modals .immersive-reader-button {
+    background-image: @immersiveReaderIcon;
+    width: 1.8rem;
+    height: 1.8rem;
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-color: transparent;
+    box-shadow: none;
+    transition: none;
+    float: left;
+    margin-top: 10px;
+}
+
+.hc .modals .ui.button.immersive-reader-button {
```

Comment:
small: imo it's preferable to have all the .hc styles in highcontrast.less

Reply:
oh makes sense I'll move